packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -196,7 +196,8 @@ export default class QuanticCategoryFacet extends LightningElement {
     this.unsubscribe = this.facet.subscribe(() => this.updateState());
     registerToStore(this.engineId, Store.facetTypes.CATEGORYFACETS, {
       label: this.label,
-      facetId: this.facet.state.facetId
+      facetId: this.facet.state.facetId,
+      element: this.template.host
```

Comment:
```suggestion
      facetId: this.facet?.state?.facetId,
      element: this.template.host
```

maybe?

Reply:
oh... why do we need a reference towards the template.host now?

Reply:
I'm passing the template.host here to save all the attribute of the facet in the Quantic store which give us the ability to duplicate the facets in the modal with the same values for all attributes to get the exact same facets.